packages/go/slices/slices.go

```diff
@@ -82,6 +82,7 @@ func UniqueBy[T any, U comparable](slice []T, fn func(T) U) []T {
 	return out
 }
 
+// TODO delete this function, use utils.Contains instead
```

Comment:
Is this something to cleanup before a merge?

Reply:
I'm not really sure what this TODO means in this case. Afaik, we don't have a `utils.Contains` function, if we ever did it was moved to this package to move us away from a collect-all utils package. Furthermore, if we were to move away from this implementation, we would almost certainly want to switch to the stdlib `slices` package which has its own implementation of Contains and most of our other utilities in this current package. We will have to wait until we adopt Go 1.21 though

Reply:
Regardless, the utils package is considered "deprecated", so if anything should move it's things in `utils` moving to more distinct packages. In this case though, we're planning on migrating our usage of our `slices` package to the newer stdlib `slices` package to reduce maintenance, so we're going to be moving away from both this package and utils wherever possible.